edbterraform/data/terraform/aws/modules/specification/variables.tf

```diff
@@ -80,13 +80,15 @@ variable "spec" {
         size_gb   = number
         iops      = optional(number)
         encrypted = optional(bool)
+        fs        = optional(string)
```

Comment:
If we could have this expanded out to `filesystem` or similar.

Reply:
done, forced push.